config.schema.yaml

```diff
@@ -57,4 +57,85 @@
     description: "Project to be used as default for actions that don't require or allow a project"
     type: "string"
     secret: false
-    required: true
+    required: false
+  default_profile:
+    description: "Select which profile will be default. If left blank will use the standard Jira connection details"
+    type: "string"
+    secret: false
+    required: false
+  profiles:
+    description: "Jira Connection profile."
+    type: "array"
+    required: false
+    items:
+      type: "object"
+      required: false
+      additionalProperties: false
+      properties:
```

Comment:
Instead of duplicating all of the schema for the inline config values, you should be able to use [YAML anchors and extend](https://blog.daemonl.com/2016/02/yaml.html) to minimize the amount of schema you need to write.

Something like this (leave the comments in for the next person who modifies the schema):

`config.schema.yaml`:
```yaml
---
# Use YAML anchors to anchor the root document
# See: https://blog.daemonl.com/2016/02/yaml.html
<<: &root_anchor
  # All indented keys here will be included when referencing root_anchor
  url:
    # ...
  poll_interval:
    # ...
  project:
    # ...
# Dedented keys will not be included when referencing root_anchor
profiles:
  description: Different Jira profiles that users can specify
  type: object
  required: false
  # No specification on dictionary *key* names
  properties: {}
  # Specify the schema for dictionary *values*
  # See the section on additionalProperties from:
  # https://json-schema.org/understanding-json-schema/reference/object.html
  additionalProperties:
    type: object
    additionalProperties: false
    properties:
      # Reuse the properties from the document root
      <<: *root_anchor
```

that YAML gets interpreted into this:

```json
{
  "url": {
    # ...
  },
  "poll_interval": {
    # ...
  },
  "project": {
    # ...
  },
  "profiles": {
    "description": "Different Jira profiles that users can specify",
    "type": "object",
    "required": false,
    "properties": {},
    "additionalProperties": {
      "type": "object",
      "additionalProperties": false,
      "properties": {
        "url": null,
        "project": {
          # ...
        },
        "poll_interval": {
          # ...
        }
      }
    }
  }
}
```

Reply:
Anchor has been used